client/src/js/collections/DogsCollection.js

```diff
@@ -5,17 +5,30 @@ var Dog = Backbone.Model.extend({
     display_id: 0,
     name: '',
     checked_in: false,
-    avatar: ''
+    avatar: '',
+    class: ''
```

Comment:
What is this new `class` attribute for? I can't find it being used anywhere else.